content/guides/getting-started/component-framework-configuration.md

```diff
@@ -36,10 +36,11 @@ Configure the dev server to use the same Webpack configuration used by Create
 React App. You can do this using the `react-scripts` plugin provided by the
 `@cypress/react` module.
 
-Also, you need to tell Cypress where to find your component tests. The following
-example configuration assumes that all the test files are somewhere in the `src`
-folder and end with a `.test.js`, `.test.jsx`, `.test.ts` or `.test.tsx`
-extension.
+By default, Cypress looks for spec files anywhere in your project with an
```

Comment:
You need to make an equivalent change to the text preceding the 4 other examples where `specPattern` is used. People are most likely going to jump to the example that is specific to their needs, and won't see this one. Double-check the examples at https://github.com/cypress-io/cypress-component-testing-examples to figure out what the best non-default `specPattern` value should be for each example.

Reply:
@cowboy We can chat when you review the latest if need be. There's some nuance where I kept the Next.js (Webpack 4) example mostly in tact while it's testing an index component in the `pages` directory. But we also have a note in the Nuxt section about that sort of content being better used as part of e2e tests. I left these in and we can talk through if changes are required to make things more clear.